beets/autotag/__init__.py

```diff
@@ -40,6 +40,7 @@
     'album': (
         'va',
         'releasegroup_id',
+        'release_group_title',
```

Comment:
I think we don't need this—if we leave this off, this data will get applied automatically.